app/form_objects/support/bulk_allocation_form.rb

```diff
@@ -25,17 +25,45 @@ def batch_id
 
 private
 
+  def add_school_to_vcap(school)
+    vcaps[school.responsible_body_id] += 1
+  end
```

Comment:
We should consider using more descriptive names here so that it is clear that it is not adding to an actual `vcap`.
I'm not 100% sure what the name should be, but something along the lines of `add_school_for_post_processing_of_vcaps`. Also, the `vcaps` accessor method and the `vcap` hash itself should also be a bit more descriptive.

Reply:
Renamed